src/tutorial.rs

```diff
@@ -534,24 +534,24 @@ When using [`Split`], keep in mind its limitations:
 - It's a type error to [`Send`] or [`Choose`] on the receive-only end.
 - It's a type error to [`Recv`] or [`Offer`] on the transmit-only end.
 - It's a runtime [`SessionIncomplete`] error if you don't drop both the `tx` and `rx` ends
-  before the future completes. This is subject to the same behavior as in [`seq`](Chan::seq),
+  before the future completes. This is subject to the same behavior as in [`call`](Chan::call),
   described below. [See here for more explanation](#errors-in-subroutines-what-not-to-do).
 
 # Sequencing and Modularity
 
-The final session type provided by Dialectic is the [`Seq`] type, which permits a more modular
+The final session type provided by Dialectic is the [`Call`] type, which permits a more modular
 way of constructing session types and their implementations. At first, you will likely not need
-to use [`Seq`]; however, as you build larger programs, it makes it a lot easier to split them up
+to use [`Call`]; however, as you build larger programs, it makes it a lot easier to split them up
 into smaller, independent specifications and components.
 
-So, what does it do? A session type `Seq<P, Q>` means "run the session `P`, then run the session
+So, what does it do? A session type `Call<P, Q>` means "run the session `P`, then run the session
 `Q`". You can think of it like the `;` in Rust, which concatenates separate statements. In
```

Comment:
Might be nice to clarify that "run the session `P`, then run the session `Q`" means to run the session `P` until it is `Done`, given that just how most session types have an "and then" type. That's the bit that confused me at first. Maybe change to ```A session type `Call<P, Q>` means "run the session `P`, then once `P` is `Done`, run the session `Q`"```?

Reply:
Great suggestion! Resolved with your wording.